barretenberg/cpp/src/barretenberg/stdlib/primitives/bigfield/README.md

```diff
@@ -347,13 +347,13 @@ This means that the carries $\textcolor{skyblue}{C_{\textsf{low}}}$ and $\textco
 > X_{\textsf{high}} = C_{\textsf{high}} \rmul \const{2^{2L}}
 > =&\ C_{\textsf{low}} \ \radd \
 > \color{brown}{\boxed{
-> (a_0 \rmul b_2 \ \radd \ a_2 \rmul b_0 - r_2) \ \radd \ (a_0 \rmul b_3 \ \radd \ a_3 \rmul b_0 \ \rsub \ r_3) \rmul \const{2^L}
```

Comment:
The before and after here are mathematically equivalent but, due to the nature of our constraints, `r_2` is actually defined directly as part of `h_1` rather than `h_0`